src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilterType } from '../../types/FilterType';
+import { Todo } from '../../types/Todo';
+import { filterTodos } from '../../utils/helpers';
+
+interface Props {
+  filterType: FilterType,
+  onFilterSelect: (filterT: FilterType) => void,
+  todos: Todo[],
+  clearCompleted: () => void
+}
+
+export const Footer: React.FC<Props> = ({
```

Comment:
The `Filter` component copies the functionality of the `Footer` component, what's the point?